src/util/yaksu_pool.c

```diff
@@ -77,6 +77,7 @@ int yaksu_pool_free(yaksu_pool_s pool)
                 count++;
 
         pool_head->free_fn(tmp->slab);
+        pool_head->free_fn(tmp->elems);
```

Comment:
```suggestion
        free(tmp->elems);
```

Reply:
I need to go back and see why my valgrind testing didn't catch any of these errors.  These are fairly obvious errors that valgrind should be able to catch.

Reply:
AddressSanitizer is a wonder. I think `libfabric` enables it be default for debug configurations. Probably a good idea for new projects.

Reply:
I'll add it to yaksa's testing too.